dbt/adapters/databricks/relation.py

```diff
@@ -1,12 +1,22 @@
 from dataclasses import dataclass
 
-from dbt.exceptions import RuntimeException
-
+from dbt.adapters.base.relation import Policy
 from dbt.adapters.spark.relation import SparkRelation
 
 
+@dataclass
+class DatabricksIncludePolicy(Policy):
+    database: bool = False  # TODO: should be True
```

Comment:
If this is `True`, some integration tests fail. Need further investigation in the following PRs.